.local.dic

```diff
@@ -138,8 +141,8 @@ VoiceOver
 Vue
 W3C
 WAI-ARIA
-Watchman
 WCAG
+Watchman
```

Comment:
I think this one used to be at the right position.

Reply:
The sort and filter were done using JS, see the commits' body.

Reply:
New commit which sorts the file by taking into consideration non-ASCII characters.